Jenkinsfile_CNP

```diff
@@ -243,5 +243,5 @@ withPipeline(type, product, component) {
       reportName           : "HRS API Contract Test Report AAT"
     ]
   }
-
+  syncBranchesWithMaster(['demo', 'perftest', 'ithc'])
```

Comment:
show this be wrapped in a onMaster after ('deploy') or something like that - possibly moved to line 79

Reply:
This will trigger only on master, last step.